src/data.js

```diff
@@ -0,0 +1,44 @@
+let assert = require('assert');
+let Entity = require('azure-entities');
+
+/**
+ * Entity for tracking which caches are currently
+ * requested for busting. This allows workers to check-in
+ * with purge-cache from time-to-time to see what should
+ * be purged.
+ *
+ */
+let CacheBuster = Entity.configure({
+  version:          1,
+  partitionKey:     Entity.keys.CompositeKey('provisionerId', 'workerType'),
+  rowKey:           Entity.keys.StringKey('cacheName'),
```

Comment:
Just use a static file here... That seems fine to me... see comment for procfile..